{"uuid":"2c3d69c9-0723-4d0a-8674-90ca037b8ed8","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["dd613154-8145-4863-8b8d-451fa098036e"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687893245434,"stop":1687893245434},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687893245611,"stop":1687893245611},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687893245690,"stop":1687893245690},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687893245725,"stop":1687893245725},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687893245732,"stop":1687893245732}],"start":1687893245348,"stop":1687893245732}